### DISCUSSION TITLE
Connecting a UE to a gNodeB without knowing its arfcn

### DISCUSSION THREAD

**Discussion by Carcalmoa** [srsran/srsRAN_Project#805] [General Help] [answered]
Is it possible to get information from the UE about all the base stations present in a specific frequency range, or even in a whole band, and then connect to the gNB that provides the best conditions?

**Comment by hudouseko** [comment, marked as answer]
Hi @Carcalmoa , that's actually roughly how UE works normally - it doesn't have to know arfcn of a cell, but rather "scans" for it. For details see e.g. _[3GPP 25.304](https://portal.3gpp.org/desktopmodules/Specifications/SpecificationDetails.aspx?specificationId=1167) section 5.2 Cell selection and reselection in idle mode_ or search for "cell selection procedure" on internet. To see the information about neighboring cells you can use on Android applications like NetMonster or similar.
